### ISSUE TITLE
tf2.0.0 using  tf.saved_model.save save model,  java SavedModelBundle load can not find ouput opt

### ISSUE THREAD

**Issue by darrenwang00** [tensorflow/tensorflow#34306]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): centos7
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 2.0.0
- Python version: 3.6

**Describe the current behavior**
python code

    X = np.array([[0,0], [0,1], [1,0], [1,1]], 'float32')
    Y = np.array([[0], [1], [1], [0]], 'int64')

    input_x = tf.keras.Input(shape=(2), name="text")
    dense1 = tf.keras.layers.Dense(64, activation='relu', name="ds1")
    dense2 = tf.keras.layers.Dense(2, name="label")

    x = dense1(input_x)
    x = dense2(x)
    pred = tf.reduce_mean(x, 1, name="prediction")

    model = tf.keras.Model(inputs=input_x, outputs=pred, name= "test")

    model.compile(loss='mean_squared_error', optimizer='adam', metrics=['binary_accuracy'])
    model.fit(X, Y, batch_size=1, epochs=10, verbose=0)

    print('inputs: ', [input.op.name for input in model.inputs])
    print('outputs: ', [output.op.name for output in model.outputs])

    tf.saved_model.save(model, "./model_00")

[java]
           String modelpath="./model_00"
            SavedModelBundle bundle = SavedModelBundle.load(modelpath,"serve");
            Graph graph = bundle.graph();
            
            Iterator<Operation> iter = graph.operations();
            while (iter.hasNext()) {
                Operation opt = iter.next();
                System.out.println(opt.name());
            }

can not find  output opt ??

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
 Thanks!
